index.js

```diff
@@ -322,6 +322,13 @@ fastifyOauth2.EPIC_GAMES_CONFIGURATION = {
   tokenPath: '/epic/oauth/v1/token'
 }
 
+fastifyOauth2.YANDEX_CONFIGURATION = {
+  authorizeHost: 'https://oauth.yandex.com',
+  authorizePath: '/authorize',
+  tokenHost: 'https://oauth.yandex.com',
+  tokenPath: '/token'
```

Comment:
Does it have a revoke url?

Reply:
Yeap, it has.

POST request to `/revoke_token ` https://yandex.ru/dev/id/doc/en/tokens/token-invalidate#cleartext 

So I can add it right now.

Reply:
And it is done)